src/preact/amp-base-element.js

```diff
@@ -0,0 +1,119 @@
+import {ActionTrust_Enum} from '#core/constants/action-constants';
+import {dict} from '#core/types/object';
+
+import {PreactBaseElement} from './base-element';
+
+export class AmpPreactBaseElement extends PreactBaseElement {
+  // todo(kvchari): confirm this can be safely moved
+  /** @override */
+  mutatedAttributesCallback() {
+    // eslint-disable-next-line local/restrict-this-access
```

Comment:
can you add this file as an override to the eslint rule? since its an amp component I don't think it needs to have this lint rule applied at all

Reply:
What is the syntax for declare a file as "unrestricted"? Not immediately obvious from looking at [this file](https://github.com/ampproject/amphtml/blob/main/build-system/eslint-rules/restrict-this-access.js)

Reply:
disabling for the whole file since it's not configurable. Can make it configurable in the future if necessary but this might be a rare need